app/utils/simplified-account/format/format-card-types.js

```diff
@@ -0,0 +1,72 @@
+const formatLabel = (card) => {
+  if (card.brand === 'visa' || card.brand === 'master-card') {
+    return `${card.label} ${card.type.toLowerCase()}`
+  } else {
+    return card.brand === 'jcb' ? card.label.toUpperCase() : card.label
+  }
+}
+
+const formatCardTypesForAdminTemplate = (allCards, acceptedCards, account) => {
+  const cardDataChecklistItem = (card) => {
+    return {
+      value: card.id,
+      text: formatLabel(card),
+      checked: acceptedCards.filter(accepted => accepted.id === card.id).length !== 0,
+      requires3ds: card.requires3ds
+    }
+  }
+  const disableCheckboxIf3dsRequiredButNotEnabled = (cardTypeChecklistItem) => {
+    if (cardTypeChecklistItem.requires3ds && !account.requires3ds) {
+      cardTypeChecklistItem.disabled = true
+      cardTypeChecklistItem.hint = {
+        html: account.type === 'test' ? `${cardTypeChecklistItem.text} is not available on test accounts` : `${cardTypeChecklistItem.text} cannot be used because 3D Secure is not available. Please contact support`
+      }
+    }
+    return cardTypeChecklistItem
+  }
+  const addHintForAmexAndUnionpay = (cardTypeChecklistItem) => {
+    if (['American Express', 'Union Pay'].includes(cardTypeChecklistItem.text)) {
+      if (account.paymentProvider === 'worldpay') {
```

Comment:
this could be an `if( A && B )` rather than nested if statements

Reply:
done